client/verta/verta/_internal_utils/_utils.py

```diff
@@ -353,7 +352,6 @@ def get_default_workspace(self):
         else:  # old backend
             return self.get_personal_workspace()
 
-
```

Comment:
Sorry to trouble you, but could we keep the blank lines that were removed in this file? [The convention in Python is](https://pep8.org/#blank-lines)

> Blank Lines
>
> Surround top-level function and class definitions with two blank lines.

Reply:
reverted